deployment/build-and-stage.yaml

```diff
@@ -27,23 +27,23 @@ steps:
 - name: gcr.io/cloud-builders/gcloud
   entrypoint: bash
   args:
-    - "-c"
-    - |
-      read current_build _ <<< $(gcloud builds list --filter=tags="build-and-stage" --format="value(id,create_time)" --sort-by=create_time --ongoing)
+  - '-c'
+  - |
+    read current_build _ <<< $(gcloud builds list --filter=tags="build-and-stage" --format="value(id,create_time)" --sort-by=create_time --ongoing)
+    if [[ -z "${current_build}" ]]; then
+      echo "Could not find any matching builds."
+      exit 1
+    fi
+    while [[ "${current_build}" != "${BUILD_ID}" ]]; do
+      echo "Waiting for build ${current_build}"
+      sleep 30
+      read current_build _ <<< $(gcloud builds list --filter=tags="build-and-stage" --format="value(id,create_time)" --sort-by=create_time  --ongoing)
```

Comment:
Same here:

You could remove `create_time` from the format and then you wouldn't need to have the awkward looking underscore as part of the `read`, right?